src/wp-activate.php

```diff
@@ -127,10 +127,10 @@ function wpmu_activate_stylesheet() {
 	<div class="wp-activate-container">
 	<?php if ( ! $key ) { ?>
 
-		<h2><?php _e( 'Activation Key Required' ); ?></h2>
-		<form name="activateform" id="activateform" method="post" action="<?php echo network_site_url( $blog_details->path . 'wp-activate.php' ); ?>">
+		<h2><?php esc_html_e( 'Activation Key Required' ); ?></h2>
```

Comment:
WordPress core intentionally does not escape translations as there is a review process for the translation of strings.

```suggestion
		<h2><?php _e( 'Activation Key Required' ); ?></h2>
```